pkg/apis/v1alpha2/aksnodeclass_status.go

```diff
@@ -21,8 +21,13 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+const (
+	ConditionTypeNodeImageReady  = "NodeImageReady"
+	ConditionTypeK8sVersionReady = "K8sVersionReady"
```

Comment:
Avoid `K8sVersionReady` here too IMO.
```suggestion
	ConditionTypeKubernetesVersionReady = "KubernetesVersionReady"
```

Reply:
Updated to `Kubernetes` as well